.travis.yml

```diff
@@ -0,0 +1,16 @@
+services:
+- docker
+
+before_script:
+  - docker build --target test --tag test .
+
+script:
+  - docker run test tests
+  - docker run test tests_integration
+  - >
+    docker run
+    -e TRELLO_API_BASE_URL
+    -e TRELLO_API_KEY
+    -e TRELLO_API_TOKEN
+    -e TRELLO_BOARD_ID
```

Comment:
You don't actually need to pass this one in because the test creates a test board and sets this based on the test board's id.

Reply:
Removed trello_board_id from .travis.yml